sdk/ai/Azure.AI.Projects/src/Generated/AuthenticationType.cs

```diff
@@ -15,6 +15,8 @@ public enum AuthenticationType
         /// <summary> Entra ID authentication (formerly known as AAD). </summary>
         EntraId,
         /// <summary> Shared Access Signature (SAS) authentication. </summary>
-        SAS
+        SAS,
+        /// <summary> No authentication. </summary>
+        None
```

Comment:
Can we add a sample for `ConnectionType.AzureBlobStorage` and see if it's working?

Reply:
This is not a priority at the moment, since we are changing the way we handle storage. This PR should fix the deserialization issue (I'll verify later), but still, you won't be able to get the properties of a storage connection and won't be able to get the credentials for this connection. The service will return 404 NotFound error. A different set of Azure ML REST API is needed for that, but we agreed with the PMs that there are higher prioritize at the moment. No customer has reported this bug so far.

Reply:
I'll verify the fix manually. This is not a common usage case, so I don't think a sample is a priority at the moment.